src/main/java/uk/gov/hmcts/reform/hois/domain/entities/DecisionCommunication.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.reform.hois.domain.entities;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.Data;
+
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+@Data
```

Comment:
if you use the Data annotation then you can skip all the getters and setters boilerplate code.

Reply:
Tried it but lambok and jackson does not work together. Working on it